api_v3/lib/KalturaBaseUserService.php

```diff
@@ -145,10 +145,10 @@ protected function resetPasswordImpl($email, $linkType = KalturaResetPassLinkTyp
 		$this->validateRequestsAmount($email);
 		
 		try {
-			$new_password = UserLoginDataPeer::resetUserPassword($email, $linkType);
-			if (!$new_password)
+			$ret = UserLoginDataPeer::resetUserPassword($email, $linkType);
+			if($ret!==true)

```

Comment:
fix spacing
also in case the value is false you are not longer throwing an exception for KalturaLog::err(KalturaErrors::LOGIN_DATA_NOT_FOUND . ': ' . $email); like we used to did
you want different handling to boolean rater then string

Reply:
The function UserLoginDataPeer::resetUserPassword never return false, its true or throws exception